<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formula</class>
 <widget class="QDialog" name="Formula">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>263</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>495</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>595</width>
    <height>361</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>4</y>
     <width>68</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>40</y>
     <width>68</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_8">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>77</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>405</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="IDC_EDIT">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>4</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="IDC_FORMULA">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>8</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>公式</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="IDC_INFO">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>37</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="IDC_5">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>107</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_MAX">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>238</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_LPARENTHESIS">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>206</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_0">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>173</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_1">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>140</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_4">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>107</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_7">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>77</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_AVE">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>238</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ave</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_DOT">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>173</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_6">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>107</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_3">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>140</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_DAT">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>206</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dat</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_9">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>77</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_POWER">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>206</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>x^y</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_SUM">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>238</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sum</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_MINUS">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>140</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_PLUS">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>173</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_MUTIPLY">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>107</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_DIDVDE">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>77</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_XOR">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>140</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Xor</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_GEQUAL">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>206</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;=</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_NOT">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>173</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Not</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_AND">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>77</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>And</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_EQUAL">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>140</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>=</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_MAXT">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>238</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MaxT</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_LEQUAL">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>206</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;=</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_OR">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>107</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Or</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_MINT">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>238</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MinT</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_NEQUAL">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>173</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>!=</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_GREATE">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>77</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_LOWER">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>107</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_1HOUR">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>173</y>
     <width>33</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1Hour</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_1MON">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>238</y>
     <width>33</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1Mon</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_5MIN">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>107</y>
     <width>33</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5Min</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_15MIN">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>33</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>15Min</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_1MIN">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>77</y>
     <width>33</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1Min</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_1DAY">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>206</y>
     <width>33</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1Day</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>77</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>站名</string>
    </property>
   </widget>
   <widget class="QComboBox" name="IDC_STATION">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>5</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>间隔</string>
    </property>
   </widget>
   <widget class="QComboBox" name="IDC_COMBOMODE">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>5</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>类型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="IDC_ATTRIB">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>35</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="IDC_TYPE">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>35</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>属性</string>
    </property>
   </widget>
   <widget class="QListWidget" name="IDC_LIST">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>65</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="IDC_10">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>173</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>$</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_MIN">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>238</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_2">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>140</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_RPARENTHESIS">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>206</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>)</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
